server/controllers/calendarController.ts

```diff
@@ -17,51 +17,47 @@ export default class CalendarController {
 
     const { calendarService, notificationService, userAuthenticationService, signInService } = app.get('DataServices')
 
-    try {
```

Comment:
controllers don't need `try` blocks anymore, since error will be handled automatically